src/pages/academy/components/video-banner/_DBanner.js

```diff
@@ -70,12 +70,17 @@ const StyledDot = styled.img`
     margin: 0 10px 4px;
 `
 
-const Dbanner = ({ video_list }) => {
+const Dbanner = ({ featured_video_list, non_featured_video_list }) => {
     const [show, setShow] = useState(false)
     const handleCloseVideo = () => setShow(false)
     const handleOpenVideo = () => setShow(true)
-    const featured_video = video_list.find((v) => v.featured)
-    const filtered_video = video_list.filter((v) => v !== featured_video)
+
+    let featured_video
+    if (featured_video_list && featured_video_list.length) {
+        featured_video = featured_video_list[0]
+    } else {
+        featured_video = non_featured_video_list[0]
+    }
```

Comment:
```suggestion
    } else {
        featured_video = non_featured_video_list[0]
        non_featured_video_list.shift()
```

Reply:
We'll need to do remove the first item of the non featured list if we assign it as the featured video during the fallback case.